domains/chiragnahata.json

```diff
@@ -0,0 +1,10 @@
+{
+    "description": "Chirag Nahata Website",
+    "owner": {
+        "username": "chiragnahata",
+        "email": "chiragnahata05@gmail.com"
+    },
+    "record": {
+        "A": "129.213.151.29"
```

Comment:
Should be an array of strings.

Reply:
{
    "domain": "is-a.dev",
    "subdomain": "chiragnahata",

    "owner": {
        "email": "chiragnahata05@gmail.com"
    },

    "records": {
        "A": ["129.213.151.29"]
    },